src/app/features/ledger/flows/bitcoin-tx-signing/ledger-bitcoin-sign-tx-container.tsx

```diff
@@ -106,6 +107,13 @@ function LedgerSignBitcoinTxContainer() {
       },
     });
 
+  function closeAction() {
+    appEvents.publish('ledgerBitcoinTxSigningCancelled', {
+      unsignedPsbt: unsignedTransaction ? bytesToHex(unsignedTransaction.toPSBT()) : '',
+    });
```

Comment:
If the action is cancelled, why do we return anything at all with the action? Do we need this?

Reply:
we need for abort handler
```
    function signingAbortedHandler(msg: GlobalAppEvents['ledgerBitcoinTxSigningCancelled']) {
      if (msg.unsignedPsbt === psbt) {
        appEvents.unsubscribe('ledgerBitcoinTxSigningCancelled', signingAbortedHandler);
        appEvents.unsubscribe('ledgerBitcoinTxSigned', txSignedHandler);
        reject(new Error('User cancelled the signing operation'));
      }
    }
```